cmd/transcriber/config/config.go

```diff
@@ -0,0 +1,214 @@
+package config
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+	"regexp"
+	"strings"
+)
+
+var idRE = regexp.MustCompile(`^[a-z0-9]{26}$`)
+
+const (
+	// defaults
+	ModelSizeDefault     = ModelSizeBase
+	TranscribeAPIDefault = TranscribeAPIWhisperCPP
+	OutputFormatDefault  = OutputFormatVTT
+)
+
+type OutputFormat string
+
+const (
+	OutputFormatVTT OutputFormat = "vtt"
+)
+
+type ModelSize string
+
+const (
+	ModelSizeTiny   ModelSize = "tiny"
+	ModelSizeBase             = "base"
+	ModelSizeSmall            = "small"
+	ModelSizeMedium           = "medium"
+	ModelSizeLarge            = "large"
+)
+
+type TranscribeAPI string
+
+const (
+	TranscribeAPIWhisperCPP    = "whisper.cpp"
+	TranscribeAPIOpenAIWhisper = "openai/whisper"
+)
+
+type CallTranscriberConfig struct {
+	// input config
+	SiteURL         string
+	CallID          string
```

Comment:
We could but then we'd need to make breaking changes to the recorder job as well. So it's either consistency or clarity. 

Also I still think it will be just `CallID` in the future so I am struggling to see a problem here since it's only ever used with one meaning.

Reply:
Well, yes it's an implementation detail. Personally I think the confusion should only exist in a minority of cases where you actually need the ID of the ephemeral call state object to do some discrimination on which specific call certain events refer to. But for the majority of instances a call ID coincides with the channel ID while also allowing for future decoupling, at which point there will likely be only one call unique ID.

> I don't think a breaking change is that big of a deal

I guess it depends on where we break. If we wanted to be consistent across clients we'd have to change Desktop and Mobile which are not as easy to change.

Reply:
Decided to approach this in isolation so we can make one single change across to keep everything consistent.

https://mattermost.atlassian.net/browse/MM-54253